mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -48,13 +51,32 @@ def InternalConfigure(self, params):
+        – use_radiation_loss: if True, radiation loss will be included in the complex lineshape; should be set to True except for testing purposes
+        – resolution_function: string determinign type of resolution function; options are 'simulated_resolution', 'gaussian_resolution', 'gaussian_lorentzian_composite_resolution'.
+        – ratio_gamma_to_sigma: parameter in gaussian_lorentzian_composite_resolution; see the MultiGasComplexLineShape processor
+        – gaussian_proportion: also a parameter in gaussian_lorentzian_composite_resolution
+        – A_array: parameter for Gaussian resolution with variable width; see the MultiGasComplexLineShape processor
+        - sigma_array: also a parameter for Gaussian resolution with variable width
+        – fit_recon_eff: if True, determine recon eff parameters by fitting to FTG data, instead of by using parameters given in self.recon_eff_parameters
+        – use_combined_four_trap_inst_reso: for simulated resolution, combine resolution distributions provided for four different traps
+        - sample_ins_resolution_errors: if True, and if using a simulated instrumental resolution, the count numbers in that distribution will be sampled from uncertainties
+        - scattering_sigma [eV]: lineshape parameter - 0-th peak gaussian broadening standard deviation
+        - min_energy [eV]: minimum of lineshape energy window for convolution with beta spectrum. Same magnitude is used for the max energy of the window.
+        - efficiency_path: path to efficiency vs. frequency (and uncertainties)
         - simplified_scattering_path: path to simplified lineshape parameters
+        – path_to_osc_strengths_files: path to oscillator strength files containing energy loss distributions for the gases in self.gases
+        – path_to_scatter_spectra_file: path to scatter spectra file, which is generated from the osc strenth files and accounts for cross scattering
+        – rad_loss_path: path to file containing data describing radiation loss
         - path_to_detailed_scatter_spectra_dir: path to oscillator and or scatter_spectra_file
-        - efficiency_path: path to efficiency vs. frequency (and uncertainties)
+        – path_to_ins_resolution_data_txt: path to file containing simulated instrumental resolution data, already combined among the four traps
+        - path_to_four_trap_ins_resolution_data_txt: path to files containing simulated instrumental resolution data for each of the four individual traps
+        - final_states_file: path to file containing molecular final state binding energies and corresponding probabilities
         - use_lineshape (boolean): determines whether tritium spectrum is smeared by lineshape.
           If False, it will only be smeared with a Gaussian
         - detailed_or_simplified_lineshape: If use lineshape, this string determines which lineshape model is used.
```

Comment:
Do you think we should write in the comments what the default value is for each parameter? Or can we expect the user to scroll down and look at the config for that parameter if they need to know the default?

I agree that it would be helpful to note which parameters are required and which are optional. I will add that info in the comments.

Reply:
I would say noting which are required is sufficient. But I would like to be able to read from the processor description what the default model is: for example which resolution function and which tail effects are included by default and which aren't.